releasenotes.rb

```diff
@@ -48,6 +48,16 @@ class ReleaseNotes < Thor
 
 ## Non-Customer Facing Changes
 
+## Special Deploy Requirements
+
+<% if commits.select(&:special_deploy_requirements).empty? %>
```

Comment:
we already have a list of commits with special deploy requirements at the end. should we just add to that to show what the actual requirements are?

I like keeping them at the end, because they are really for the developer and for those who just want to know what we changed.